aii-core/src/main/perl/Shellfe.pm

```diff
@@ -464,16 +435,22 @@ sub dhcp
        }
     }
 
-    my @commands = $self->dhcpcfg ($node, $cmd, $st, $mac);
-    $self->debug (4, "Going to run: " .join (" ",@commands));
-    my $output;
-    $output = CAF::Process->new(\@commands, log => $self)->output();
-    if ($?) {
-       $self->error("Error when configuring $node: $output");
-    } else {
-        $self->debug(4, "Output: $output");
-    };
-    $self->debug(4, "End output");
+    my $ec;
+    if ("$cmd" eq CONFIGURE) {
+        my $opts = $st->{configuration}->getElement (DHCPOPTION)->getTree;
+        $self->debug (4, "Going to add dhcp entry of $node to configure");
+        $ec = $mgr->new_configure_entry($node, $mac, $opts->{tftpserver} // ';', $opts->{addoptions} // ());
```

Comment:
the `//` magic is not avail in el5, but it is in el6. so at least require perl 5.10.0.